fsspec/asyn.py

```diff
@@ -92,6 +92,50 @@ def get_loop():
     return loop[0]
 
 
+try:
+    import resource
```

Comment:
`resource` has been around a long time, why would the import fail?

Reply:
It doesn't work on windows (AFAIK)